commands/cleanupwatchdog.c

```diff
@@ -237,7 +237,7 @@ do_watchdog_disable(uint32_t *id)
 
     hypercall.op = __HYPERVISOR_sched_op;
     hypercall.arg[0] = SCHEDOP_watchdog;
-    hypercall.arg[1] = (__u64) (unsigned int) &arg;  // pointer to u64
+    hypercall.arg[1] = (__u64) (uintptr_t) &arg;  // pointer to u64
```

Comment:
Minor, the conversion to `(__u64)` is probably worthless.

Reply:
Indeed, the array field is already declared as `__u64`,  dropped these now